Mathlib/Tactic/Positivity/Finset.lean

```diff
@@ -0,0 +1,49 @@
+/-
+Copyright (c) 2023 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Data.Fintype.Card
+import Mathlib.Tactic.Positivity.Core
+
+/-!
+# Positivity extensions for finsets
+
+This file provides a few `positivity` extensions that cannot be in either the finset files (because
+they don't know about ordered fields) or in `Tactic.Positivity.Basic` (because it doesn't want to
+know about finiteness).
+-/
+
+open Finset
+namespace Mathlib.Meta.Positivity
+open Qq Lean Meta
```

Comment:
Let's visually separate the `namespace` and `open` commands. (I suppose there is no reason to have one `open` inside and another `open` outside the namespace?)
```suggestion
namespace Mathlib.Meta.Positivity

open Qq Lean Meta Finset
```